src/seedu/addressbook/data/person/ReadOnlyPerson.java

```diff
@@ -82,4 +82,15 @@ default String getAsTextHidePrivate() {
         }
         return builder.toString();
     }
+
+    /**
+     * Returns a concatenated version of the printable strings of each object.
+     */
+    default String getPrintableString(Printable... printables){
+        final StringBuilder builder = new StringBuilder();
```

Comment:
Good job with using StringBuilder, another class that serves this purpose is StringJoiner.

Reference: https://docs.oracle.com/javase/8/docs/api/java/util/StringJoiner.html